### ISSUE TITLE
 Error running Gradle: ProcessException: Process "/home/hp/flutter-project/hello_world/android/gradlew" exited abnormally: Starting a Gradle Daemon, 1 busy Daemon could not be reused, use --status for details

### ISSUE THREAD

**Issue by roshanaryal** [flutter/flutter#41691]
i am using ubuntu as OS and try to run first flutter app but got this error can anyone help me
error
```
Launching lib/main.dart on 192.168.56.101:5555 in debug mode...
* Error running Gradle:
ProcessException: Process "/home/hp/flutter-project/hello_world/android/gradlew" exited abnormally:
Starting a Gradle Daemon, 1 busy Daemon could not be reused, use --status for details

The message received from the daemon indicates that the daemon has disappeared.
Build request sent: Build{id=77391d2d-395d-43ac-b46a-479023678702, currentDir=/home/hp/flutter-project/hello_world/android}
Attempting to read last messages from the daemon log...
Daemon pid: 18834
  log file: /home/hp/.gradle/daemon/4.10.2/daemon-18834.out.log
----- Last  20 lines from daemon log file - daemon-18834.out.log -----
19:29:17.426 [ERROR] [org.gradle.launcher.daemon.server.DaemonStateCoordinator] Command execution: started DaemonCommandExecution[command = Build{id=77391d2d-395d-43ac-b46a-479023678702, currentDir=/home/hp/flutter-project/hello_world/android}, connection = DefaultDaemonConnection: socket connection from /0:0:0:0:0:0:0:1:42939 to /0:0:0:0:0:0:0:1:40482] after 0.0 minutes of idle
19:29:17.427 [INFO] [org.gradle.launcher.daemon.server.DaemonRegistryUpdater] Marking the daemon as busy, address: [458688ff-7357-40eb-a514-40772633dd44 port:42939, addresses:[/0:0:0:0:0:0:0:1%lo, /127.0.0.1]]
19:29:17.427 [DEBUG] [org.gradle.launcher.daemon.registry.PersistentDaemonRegistry] Marking busy by address: [458688ff-7357-40eb-a514-40772633dd44 port:42939, addresses:[/0:0:0:0:0:0:0:1%lo, /127.0.0.1]]
19:29:17.428 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire exclusive lock on daemon addresses registry.
19:29:17.428 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
19:29:17.430 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
19:29:17.431 [DEBUG] [org.gradle.launcher.daemon.server.DaemonStateCoordinator] resetting idle timer
19:29:17.431 [DEBUG] [org.gradle.launcher.daemon.server.DaemonStateCoordinator] daemon is running. Sleeping until state changes.
19:29:17.433 [INFO] [org.gradle.launcher.daemon.server.exec.StartBuildOrRespondWithBusy] Daemon is about to start building Build{id=77391d2d-395d-43ac-b46a-479023678702, currentDir=/home/hp/flutter-project/hello_world/android}. Dispatching build started information...
19:29:17.434 [DEBUG] [org.gradle.launcher.daemon.server.SynchronizedDispatchConnection] thread 16: dispatching class org.gradle.launcher.daemon.protocol.BuildStarted
Connection] thread 16: dispatching class org.gradle.launcher.daemon.protocol.BuildStarted
19:29:17.438 [DEBUG] [org.gradle.launcher.daemon.server.exec.EstablishBuildEnvironment] Configuring env variables: {PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/home/hp/bin:/usr/lib/jvm/jdk1.8.0_212/bin:/home/hp/bin:/usr/lib/jvm/jdk1.8.0_212/bin:home/hp/flutter/bin:home/hp/tools/bin, XAUTHORITY=/run/user/1000/gdm/Xauthority, FLUTTER_HOST=VSCode, XMODIFIERS=@im=ibus, GDMSESSION=ubuntu, XDG_DATA_DIRS=/usr/share/ubuntu:/usr/local/share/:/usr/share/:/var/lib/snapd/desktop, TEXTDOMAINDIR=/usr/share/locale/, GTK_IM_MODULE=ibus, VSCODE_LOG_STACK=false, PUB_ENVIRONMENT=vscode.dart-code, DBUS_SESSION_BUS_ADDRESS=unix:path=/run/user/1000/bus, VSCODE_PID=10081, XDG_CURRENT_DESKTOP=Unity, SSH_AGENT_PID=1743, QT4_IM_MODULE=xim, SESSION_MANAGER=local/hp-HP-350-G1:@/tmp/.ICE-unix/1666,unix/hp-HP-350-G1:/tmp/.ICE-unix/1666, USERNAME=hp, LOGNAME=hp, APPLICATION_INSIGHTS_NO_DIAGNOSTIC_CHANNEL=true, PWD=/home/hp/flutter-project/hello_world/android, IM_CONFIG_PHASE=2, DERBY_HOME=/usr/lib/jvm/jdk1.8.0_221/db, VSCODE_NLS_CONFIG={"locale":"en-us","availableLanguages":{},"_languagePackSupport":true}, AMD_ENTRYPOINT=vs/workbench/services/extensions/node/extensionHostProcess, GJS_DEBUG_TOPICS=JS ERROR;JS LOG, SHELL=/bin/bash, CHROME_DESKTOP=code-url-handler.desktop, GIO_LAUNCHED_DESKTOP_FILE=/usr/share/applications/code.desktop, J2SDKDIR=/usr/lib/jvm/jdk1.8.0_221, OLDPWD=/home/hp/flutter-project/hello_world/android, VSCODE_PREVENT_FOREIGN_INSPECT=true, GNOME_DESKTOP_SESSION_ID=this-is-deprecated, GTK_MODULES=gail:atk-bridge, J2REDIR=/usr/lib/jvm/jdk1.8.0_221/jre, VSCODE_NODE_CACHED_DATA_DIR=/home/hp/.config/Code/CachedData/b37e54c98e1a74ba89e03073e5a3761284e3ffb0, CLUTTER_IM_MODULE=xim, TEXTDOMAIN=im-config, VERBOSE_LOGGING=true, XDG_SESSION_DESKTOP=ubuntu, SHLVL=2, VSCODE_HANDLES_UNCAUGHT_ERRORS=true, QT_IM_MODULE=ibus, JAVA_HOME=/snap/android-studio/78/android-studio/jre, FLUTTER_ROOT=/home/hp/flutter, XDG_CONFIG_DIRS=/etc/xdg/xdg-ubuntu:/etc/xdg, LANG=en_US.UTF-8, XDG_SESSION_TYPE=x11, XDG_SESSION_ID=2, PIPE_LOGGING=true, DISPLAY=:0, ELECTRON_RUN_AS_NODE=1, LIBVIRT_DEFAULT_URI=qemu:///system, VSCODE_IPC_HOOK_EXTHOST=/tmp/vscode-ipc-d5fd87cc-8e8d-4cd3-91fe-f554552a5564.sock, FLUTTER_SUPPRESS_ANALYTICS=true, GPG_AGENT_INFO=/run/user/1000/gnupg/S.gpg-agent:0:1, DESKTOP_SESSION=ubuntu, VSCODE_LOGS=/home/hp/.config/Code/logs/20191001T183454, USER=hp, XDG_MENU_PREFIX=gnome-, GIO_LAUNCHED_DESKTOP_FILE_PID=10081, WINDOWPATH=2, QT_ACCESSIBILITY=1, GJS_DEBUG_OUTPUT=stderr, XDG_SEAT=seat0, SSH_AUTH_SOCK=/run/user/1000/keyring/ssh, NO_AT_BRIDGE=1, GNOME_SHELL_SESSION_MODE=ubuntu, XDG_RUNTIME_DIR=/run/user/1000, XDG_VTNR=2, HOME=/home/hp, VSCODE_IPC_HOOK=/run/user/1000/vscode-fbac0697-1.38.1-main.sock}
19:29:17.450 [DEBUG] [org.gradle.launcher.daemon.server.exec.LogToClient] About to start relaying all logs to the client via the connection.
19:29:17.450 [INFO] [org.gradle.launcher.daemon.server.exec.LogToClient] The client will now receive all logging from the daemon (pid: 18834). The daemon log file: /home/hp/.gradle/daemon/4.10.2/daemon-18834.out.log
19:29:17.452 [INFO] [org.gradle.launcher.daemon.server.exec.LogAndCheckHealth] Starting build in new daemon [memory: 1.4 GB]
19:29:17.453 [DEBUG] [org.gradle.launcher.daemon.server.SynchronizedDispatchConnection] thread 14: received class org.gradle.launcher.daemon.protocol.CloseInput
19:29:17.453 [DEBUG] [org.gradle.launcher.daemon.server.DefaultDaemonConnection] thread 14: Received IO message from client: org.gradle.launcher.daemon.protocol.CloseInput@4edae195
19:29:17.464 [INFO] [org.gradle.launcher.daemon.server.exec.ForwardClientInput] Closing daemon's stdin at end of input.
19:29:17.464 [INFO] [org.gradle.launcher.daemon.server.exec.ForwardClientInput] The daemon will no longer process any standard input.
19:29:17.482 [DEBUG] [org.gradle.launcher.daemon.server.exec.ExecuteBuild] The daemon has started executing the build.
19:29:17.483 [DEBUG] [org.gradle.launcher.daemon.server.exec.ExecuteBuild] Executing build with daemon context: DefaultDaemonContext[uid=6e09c411-c05d-4af7-a308-c02b0bb7b4d4,javaHome=/snap/android-studio/78/android-studio/jre,daemonRegistryDir=/home/hp/.gradle/daemon,pid=18834,idleTimeout=10800000,daemonOpts=-Xmx1536M,-Dfile.encoding=UTF-8,-Duser.country=US,-Duser.language=en,-Duser.variant]
----- End of the daemon log -----


FAILURE: Build failed with an exception.

* What went wrong:
Gradle build daemon disappeared unexpectedly (it may have been killed or may have crashed)

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org
  Command: /home/hp/flutter-project/hello_world/android/gradlew app:properties

Please review your Gradle project setup in the android/ folder.

```

### this is the result when i run flutter doctor -v

```
hp@hp-HP-350-G1:~$ flutter doctor -v
[✓] Flutter (Channel stable, v1.9.1+hotfix.2, on Linux, locale en_US.UTF-8)
    • Flutter version 1.9.1+hotfix.2 at /home/hp/flutter
    • Framework revision 2d2a1ffec9 (4 weeks ago), 2019-09-06 18:39:49 -0700
    • Engine revision b863200c37
    • Dart version 2.5.0

 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    • Android SDK at /home/hp/Android/Sdk
    • Android NDK location not configured (optional; useful for native profiling
      support)
    • Platform android-29, build-tools 29.0.2
    • ANDROID_HOME =
      /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/u
      sr/local/games:/snap/bin:/home/hp/bin:/usr/lib/jvm/jdk1.8.0_212/bin:/usr/l
      ib/jvm/default-java/bin:/usr/java/jdk1.8.0_212//bin:home/hp/flutter/bin:ho
      me/hp/tools/bin:home/hp
    • Java binary at: /snap/android-studio/78/android-studio/jre/bin/java
    • Java version OpenJDK Runtime Environment (build
      1.8.0_202-release-1483-b49-5587405)
    • All Android licenses accepted.

[✓] Android Studio (version 3.5)
    • Android Studio at /snap/android-studio/78/android-studio
    • Flutter plugin version 39.0.3
    • Dart plugin version 191.8423
    • Java version OpenJDK Runtime Environment (build
      1.8.0_202-release-1483-b49-5587405)

[✓] VS Code (version 1.38.1)
    • VS Code at /usr/share/code
    • Flutter extension version 3.4.1

[✓] Connected device (1 available)
    • 192.168.56.101:5555 • 192.168.56.101:5555 • android-x86 • Android 4.1.1
      (API 16)

• No issues found!
```

**Comment by BondarenkoStas** [comment]
@roshanaryal Can you please check https://github.com/jangrewe/gitlab-ci-android/issues/39#issuecomment-522076777 if that will help?

**Comment by roshanaryal** [comment]
@BondarenkoStas Thanks for your response 
i didn't understand any thing in that comment .
should i have to update gradale or what?

**Comment by blasten** [comment]
What @BondarenkoStas said. This issue seems to be fixed in 5.3 https://github.com/gradle/gradle/issues/3117#issuecomment-474137063

**Comment by janmoppel** [comment]
Hello @roshanaryal !

Without additional information, we are unfortunately not sure how to proceed on this issue. We are therefore reluctantly going to close this bug for now. Please don't hesitate to comment on the issue, if you have any more information for us; we will reopen it right away!

Thank you for your contribution.

P.S. If anyone still experience the same problem, please file a new issue with the information required in the template.
All system setups can be slightly different so it’s always better to open new issues and reference related issues.
